conf/sample.ini

```diff
@@ -551,6 +551,21 @@
 # Syslog tag. By default, the process' argv[0] is used.
 ;tag =
 
+[log.frontend]
+
```

Comment:
Nit, but could you drop this newline? We don't have them after section headers elsewhere in this file.

Reply:
- [ ] remove whitespace